### PR TITLE
NETOBSERV-1816 allow n/a filter

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

Allow filtering on `n/a` by searching on strict value `""` in the text input.

image
image
image
image

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [X] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: web/src/utils/filter-definitions.ts [hunk 1]
```diff
@@ -38,6 +38,8 @@ import { validateK8SName, validateStrictK8SName } from './label';
 
 // Convenience string to filter by undefined field values
 export const undefinedValue = '""';
+// Unique double are allowed while typing but invalid
```
[the remark this answers is not in the document]
Comment (jpinsonneau) [reply web/src/utils/filter-definitions.ts:41]: that should do the trick @jotak 
https://github.com/netobserv/network-observability-console-plugin/pull/590/commits/14617a00cf179cad2c8e0c863e0cad738ca40899

**Comment by jotak** [comment]
👌
/lgtm

**Comment by jpinsonneau** [comment]
> @jpinsonneau I now see all filters now accept `""` to filter on `n/a`. So that looks good. But I tried greater than with `n/a` to see how it behaves, it shows error; but maybe we can handle the error better? wdyt? Screenshot 2024-09-10 at 11 42 17 AM

doh

**Comment by jpinsonneau** [comment]
@Amoghrd that will fix the issue you found:
https://github.com/netobserv/network-observability-console-plugin/pull/590/commits/4664365224df3d468f158b41bb9f5501acdb4f93

Screenshot from 2024-09-10 18-05-55
Screenshot from 2024-09-10 18-06-05

**Comment by Amoghrd** [comment]
That helped. Works as expected!
/label qe-approved

**Comment by jpinsonneau** [comment]
Awesome, thanks @Amoghrd !

**Pull request merged**